Arista/ConfigFiles/telegraf-qubit-scylla-dev.conf

```diff
@@ -0,0 +1,69 @@
+###############################################################################
+#                            OUTPUT PLUGINS                                   #
+###############################################################################
+
+# Configuration for influxdb server to send metrics to
+[[outputs.influxdb]]
+  ## The full HTTP or UDP endpoint URL for your InfluxDB instance.
+  ## Multiple urls can be specified as part of the same cluster,
+  ## this means that only ONE of the urls will be written to each interval.
+  # urls = ["udp://localhost:8089"] # UDP endpoint example
+  urls = ["http://planck:8086"] # required
+  ## The target database for metrics (telegraf will create it if not exists).
+  database = "qubit_dev" # required
+  ## Retention policy to write to.
+  retention_policy = ""
+  ## Precision of writes, valid values are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+  ## note: using "s" precision greatly improves InfluxDB compression.
+  precision = "s"
```

Comment:
Do you want the precision to be seconds?

Reply:
the scrape period of these states is the default 15secs and so precision of seconds seems fine. These are not really events from Scylla